src/styles/_typography.scss

```diff
@@ -1,15 +1,41 @@
-@use "./variables" as *;
+@use "./variables"as *;
+@import url('https://fonts.googleapis.com/css2?family=Lexend+Deca:wght@400;700&display=swap');
+
 
 // here we use rem for font-size
 // rem is relative to the :root element in the dom (html tag)
 // we can manipulate the :root element's font-size at different breakpoints to scale up the font-size application-wide
 
 * {
-  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
-    "Lucida Sans", Arial, sans-serif;
+  font-family: 'Lexend Deca', sans-serif;
   color: $text-color;
 }
 
+// Splash screen = 2.5rem, 400fw 
```

Comment:
Why are these commented out here? Why are they here? We need them to be variables don't we? We don't want any commented out code ANYWHERE.  Move them to variables and delete them from here.